docs/scripts_md/updateMRI_Upload.md

```diff
@@ -0,0 +1,55 @@
+# NAME
+
+updateMRI\_Upload.pl - updates database table `mri_upload` according to an entry in table
+   tarchive
+
+# SYNOPSIS
+
+updateMRI\_Upload.pl \[options\] -profile prod -tarchivePath tarchivePath -source\_location source\_location
+
+- **-profile prod** : (mandatory) path (absolute or relative to the current directory) of the 
+    profile file
+- **-tarchivePath tarchivePath** : (mandatory) absolute path to the tarchive file
+- **-source\_location source\_location** : (mandatory) value to set column 
+    `DecompressedLocation` to for the newly created record in table `mri_upload` (see below)
+- **-globLocation** : Loosen the validity check of the tarchive allowing for the 
+     possibility that the tarchive was moved to a different directory.
```

Comment:
replace "tarchive" by "it" (or "DICOM archive")

Reply:
Done